modin/engines/base/frame/partition.py

```diff
@@ -36,7 +36,7 @@ def get(self):
         """
         pass
 
-    def apply(self, func, **kwargs):
+    def apply(self, func, *args, **kwargs):
```

Comment:
Are there any `*args` being passed here? Why do we need this change?

Reply:
I do not know the exact details but I would assume this is to support the generic way of calling arbitrary Python functions where you can pass arguments both as position and as named ones... it shouldn't be hurting much anyway.

Reply:
@devin-petersohn , yes, we have `*args` passed here for now ([link](https://github.com/modin-project/modin/pull/2919/files#diff-216a3c97cca7e07c5687d171206e7ebf2678c10bb2308f6b9fe1604468b6bed2R142)). Also, this change is necessary to pass `*args` directly to a remote call in order to a backend itself (ray/dask) is able to materialize data if the `len(call queue) == 1`. This speedups map operations. That's why I added remote `apply_func` function.

@vnlitvinov , your thoughts are on the right way as well.